migration/rules/src/main/scala/fix/v0_9_28.scala

```diff
@@ -0,0 +1,22 @@
+package fix
+
+import scala.meta._
+
+import scalafix.v1._
+
+class v0_9_28 extends SemanticRule("v0_9_28") {
+  override def fix(implicit doc: SemanticDocument): Patch =
+    doc.tree.collect {
+      case t @ Importee.Name(Name("SemanticRuleSuite")) =>
+        Patch.removeImportee(t) +
+          Patch.addGlobalImport(
+            importer"scalafix.testkit.AbstractSemanticRuleSuite"
+          )
+      case t @ Init(_, _, _) =>
+        if (t.toString() == "SemanticRuleSuite()")
+          Patch.addGlobalImport(importer"org.scalatest.FunSuiteLike") +
+            Patch.replaceTree(t, "AbstractSemanticRuleSuite") +
+            Patch.addRight(t, " with FunSuiteLike")
```

Comment:
why not
```suggestion
            Patch.replaceTree(t, "AbstractSemanticRuleSuite with FunSuiteLike")
```

Reply:
Also, I guess it does not matter since chances that `AbstractSemanticRuleSuite` was mixing-in other traits are very low, but the rule would erase the existing ones - to be correct, you could have prettyprinted a patched version of `t` instead.